### DISCUSSION TITLE
bigfix/10.0/patch/Patch/Patch_RH/c_verifying_plugin_cert_access

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#68] [Q&A]
# bigfix/10.0/patch/Patch/Patch_RH/c_verifying_plugin_cert_access

From the command line, you can use the --check-baserepos and the --check-allrepos commands to check that the entitlement certificates, which are in the 'certs' folder, have access to the supported Red Hat repositories.

https://help.hcltechsw.com/bigfix/10.0/patch/Patch/Patch_RH/c_verifying_plugin_cert_access.html

**Comment by phusitmlw** [comment]
If you have not found the command yet.
<BES Server>\DownloadPlugins\RHSMProtocol\RHSMPlugin --check-baserepos
or 
<BES Server>\DownloadPlugins\RHSMProtocol\RHSMPlugin --check-allrepos
